NEWS

```diff
@@ -51,6 +51,12 @@ CHANGES WITH 239 in spe:
           screen resolution for HiDPI systems, and now provides loader
           configuration settings to change the resolution explicitly.
 
+        * systemd-resolved now supports DNS-over-TLS ("PrivateDNS"). It's still
+          turned off by default, use PrivateDNS=yes to turn it on in
```

Comment:
I think it should be PrivateDNS=opportunistic ;) as a strict mode, with certificate verification, is not yet implemented.

Reply:
Let's fix this later.

Reply:
Please see #9282.